### PR TITLE
MAKE-1189 Update selected-tests to use poetry

### PR DESCRIPTION
**Pull request by lydiastepanek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,13 +257,13 @@ __Note__: reload is only used in development mode.
...
-black src tests
-pydocstyle src
-pytest --flake8 --isort
+poetry run black src tests
+poetry run pydocstyle src
+poetry run pytest
 ```
 
 To get code coverage information, you can run pytest directly.
+# WIP
```
Comment (jimoleary) [inline README.md]: Is this heading meant to be here?

**Review by jimoleary — COMMENTED** [review]
one question but otherwsie v lgtm!

File: README.md (hunk 1 above)
Comment (lydiastepanek) [reply README.md]: This PR is still WIP, I will assign reviewers when it is ready (and I will remove all WIP sections before that)

Comment (lydiastepanek) [reply README.md]: Now this PR is ready for review. Removed this `WIP` commend.

File: evergreen.yml [hunk 2]
```diff
@@ -37,7 +37,7 @@ tasks:
         set -o errexit
         . venv/bin/activate
 
-        pytest --flake8 --isort --junitxml=test_output_junit.xml --ignore=venv
+        poetry run pytest --flake8 --isort --junitxml=test_output_junit.xml --ignore=venv
```
Comment (dbradf) [inline evergreen.yml]: With poetry you can remove the `--ignore=venv`.

File: README.md (hunk 1 above)
Comment (dbradf) [reply README.md]: This is still showing up the diff.

**Review by dbradf — APPROVED** [review]
Can you open a MAKE ticket for this and add it to the title?

File: README.md (hunk 1 above)
Comment (lydiastepanek) [reply README.md]: Whoops, thought I had removed this. Now it's removed.

**Comment by lydiastepanek** [comment]
evergreen merge

**Pull request merged**